src/components/button.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import styles from "../styles/button.module.css"
+
+export default function PhotoStrip(props) {
```

Comment:
This component should probably be called Button haha

Reply:
🙃